test/e2e/functional/output-artifact-with-s3-bucket-creation.yaml

```diff
@@ -0,0 +1,37 @@
+# This example demonstrates bucket-creation for s3 output artifacts, when the target bucket does not exist.
+apiVersion: argoproj.io/v1alpha1
```

Comment:
maybe move to `testdata` with other test data

Reply:
done